packages/ui/src/utils/numberHelpers.ts

```diff
@@ -0,0 +1,173 @@
+import JSBI from "jsbi";
+const allCommas = /,{1}/g;
```

Comment:
`{1}` is redundant here because matching once is the default behavior of a character match in regex